site/specs/numbers.json

```
@@ -11479,47 +11502,45 @@
         }
       }
     },
-    "/accounts/{accountId}/removeImportedTnOrders/{orderid}": {
+    "/accounts/{accountId}/removeImportedTnOrders/{orderId}": {
       "get": {
         "tags": [
           "/accounts"
         ],
-        "description": "<p>Retrieve information about a removeImportedTnOrder with specified ID.</p>\n",
-        "operationId": "GET /accounts/{accountId}/removeImportedTnOrders/{orderid}",
+        "description": "Retrieve information about a removeImportedTnOrder with specified ID",
+        "summary": "Retrieve Remove Imported Tn Order",
+        "operationId": "RetrieveRemoveImportedTnOrder",
         "parameters": [
           {
             "name": "accountId",
-            "description": "",
+            "description": "User's account ID",
             "in": "path",
             "required": true,
             "schema": {
-              "type": "string",
-              "default": null
+              "type": "string"
             },
-            "example": null
+            "example": "1234567"
           },
           {
-            "name": "orderid",
-            "description": "",
+            "name": "orderId",
+            "description": "ID of Remove Imported Tn Order to retrieve",
```

Maybe this would look better?

```suggestion
            "description": "ID of `RemoveImportedTnOrder` to retrieve",
```